Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -2412,6 +2375,37 @@ theorem omega0_le_of_isLimit {o} (h : IsLimit o) : ω ≤ o :=
 @[deprecated "No deprecation message was provided."  (since := "2024-09-30")]
 alias omega_le_of_isLimit := omega0_le_of_isLimit
 
+theorem natCast_add_omega0 (n : ℕ) : n + ω = ω := by
```

Comment:
It looks like the older proof of this by induction on `n` is cleaner. What do you think?

Reply:
Said proof relied on a very ugly proof of `one_add_omega0`, which constructed an explicit order isomorphism between `ℕ` and `Unit ⊕ ℕ`, whereas this new proof relies only on ordinal arithmetic and is ultimately a bit shorter.